stream-chat-swift-docs/docs/guides/working-with-messages.md

```diff
@@ -4,17 +4,19 @@ title: Working with Messages
 
 ## Message object
 A message is represented by `ChatMessage` model.
+Depending on combination of its properties messages appear differently, this screenshot showcases the most common types of messages:
+
+<img src="../img/messages-showcase.png" width="40%" />
```

Comment:
Can we also include an image where we show which element in the message UI corresponds to which property of message? not all, just some